schemas/test/en/test_checkbox.json

```diff
@@ -24,6 +24,10 @@
         {
             "name": "ru_name",
             "type": "string"
+        },
+        {
+            "name": "ru_ref",
+            "type": "string"
         }
```

Comment:
Why was this added?.

Reply:
Added in error when i was testing the launcher PR, reverted now